File: test/export/pas/08-operadores.pas
Program sin_titulo;
Var
	a: Real;
	b: Real;
	c: Boolean;
	d: Boolean;
	e: String;
	f: String;
Begin
	a := 11;
	b := 2;
	c := True;
	d := False;
	e := 'Hola';
	f := 'Mundo';
	WriteLn(a+b);
	WriteLn(a*b);
	WriteLn(a/b);
	WriteLn(a-b);
	WriteLn(Power(a,b));
	WriteLn(a Mod b);
	WriteLn(a=b);
	WriteLn(a<>b);
	WriteLn(a<b);
	WriteLn(a>b);
	WriteLn(a<=b);
	WriteLn(a>=b);
	WriteLn(c And d);
	WriteLn(c Or d);
	WriteLn(Not (c And d));
	WriteLn(e+' '+f);
	WriteLn(e=f);
	WriteLn(e<>f);
	WriteLn(e<f);
	WriteLn(e>f);
	WriteLn(e<=f);
	WriteLn(e>=f);
End.

File: test/export/m/08-operadores.m
function r=comparar_cadenas(a,b)
	n1=length(a);
	n2=length(b);
	n=min(n1,n2);
	i=0;
	while i<n
		if (a(i)!=b(i))
			r=a(i)-b(i);
			break;
		end
		i=i+1;
	end
	if i==n
		r=n1-n2;
	end
end
function sin_titulo()
	a=11;
	b=2;
	c=true;
	d=false;
	e='Hola';
	f='Mundo';
	disp(a+b);
	disp(a*b);
	disp(a/b);
	disp(a-b);
	disp(a^b);
	disp(mod(a,b));
	disp(a==b);
	disp(a~=b);
	disp(a<b);
	disp(a>b);
	disp(a<=b);
	disp(a>=b);
	disp(c && d);
	disp(c || d);
	disp(~(c && d));
	disp([[e,' '],f]);
	disp(strcmp(e,f));
	disp(~strcmp(e,f));
	disp(comparar_cadenas(e,f)<0);
	disp(comparar_cadenas(e,f)>0);
	disp(comparar_cadenas(e,f)<=0);
	disp(comparar_cadenas(e,f)>=0);
end
